scripts/slack-notify-failure.sh

```diff
@@ -3,4 +3,4 @@
 BRANCH_NAME="$1"
 NOTIFICATION_COLOR=${2:-#EE0000}
 
-curl -X POST -H 'Content-Type:application/json' -d '{"attachments": [{"color": "'$NOTIFICATION_COLOR'", "fallback": "Build Notification: '$CIRCLE_BUILD_URL'", "title": ":warning: Kubernetes-Monitor Merge Failure :warning:", "text": ":egg_broken_1: Kubernetes-Monitor broken branch: `'$BRANCH_NAME'` :egg_broken_1:"}]}' $SLACK_WEBHOOK
+curl -X POST -H 'Content-Type:application/json' -d '{"attachments": [{"color": "'$NOTIFICATION_COLOR'", "fallback": "Build Notification: '$CIRCLE_BUILD_URL'", "title": ":warning: Kubernetes-Monitor Merge Failure :warning:", "text": ":egg_broken_1: Kubernetes-Monitor broken branch: `'$BRANCH_NAME'`:egg_broken_1:\n'$CIRCLE_BUILD_URL'"}]}' $SLACK_WEBHOOK
```

Comment:
Any reason to remove the extra space?
```shell
`'$BRANCH_NAME'` :egg_broken_1:"
```
I think it looks symmetric to the space at the start of the message 😛

Reply:
yes, the answer is in the question's body: it's an extra space
it does nothing, and is followed by a newline
why would we want it? 😬

Reply:
oh I think I got you

Reply:
yeah 🦅 👁 it looks cleaner with the space! sorry

Reply:
check again please